python312Packages.fyta-cli: 0.5.1 -> 0.6.6

fabaff opened the pull request:
Diff: https://github.com/dontinelli/fyta_cli/compare/refs/tags/v0.5.1...v0.6.2

Changelog: https://github.com/dontinelli/fyta_cli/releases/tag/v0.6.2

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
Result of `nixpkgs-review pr 335893` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.fyta</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python311Packages.fyta-cli</li>
    <li>python311Packages.fyta-cli.dist</li>
    <li>python312Packages.fyta-cli</li>
    <li>python312Packages.fyta-cli.dist</li>
  </ul>
</details>

dotlambda commented on pkgs/development/python-modules/fyta-cli/default.nix:
```diff
@@ -4,12 +4,13 @@
 buildPythonPackage rec {
   pname = "fyta-cli";
-  version = "0.5.1";
+  version = "0.6.2";
```
```suggestion
  version = "0.6.3";
```
or 0.6.6 if you want

dotlambda commented on pkgs/development/python-modules/fyta-cli/default.nix:
```diff
@@ -18,12 +19,15 @@ buildPythonPackage rec {
+  ];
 
   # Module has no tests
   doCheck = false;
```
There are now tests.

fabaff replied:
Looks like that I'm still missing a piece to get the tests running.

dotlambda commented on pkgs/development/python-modules/fyta-cli/default.nix:
```diff
@@ -4,12 +4,13 @@
+  version = "0.6.2";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```
```suggestion
  disabled = pythonOlder "3.11";
```

fabaff committed:
python312Packages.fyta-cli: 0.6.2 -> 0.6.6

Diff: https://github.com/dontinelli/fyta_cli/compare/refs/tags/v0.6.2...v0.6.6

Changelog: https://github.com/dontinelli/fyta_cli/releases/tag/v0.6.6
